iohub/cli/cli.py

```diff
@@ -87,3 +89,60 @@ def convert(input, output, grid_layout, chunks):
         chunks=chunks,
     )
     converter()
+
+
+@cli.command()
+@click.help_option("-h", "--help")
+@click.option(
+    "-i",
+    "--input",
+    type=click.Path(exists=True, file_okay=True, dir_okay=False),
+    required=True,
+    help="Path to the input Zarr file.",
+)
+@click.option(
+    "-c",
+    "--csv",
+    type=click.File("r"),
+    required=True,
+    help="Path to the CSV file containing well names.",
+)
+def rename_wells(csvfile, zarrfile):
```

Comment:
It'll be useful if this is available as a python function (i.e. not click decorated) that can be imported and used in python scripts. The CLI would then decorate and call this function, e.g.

```python
@cli.command()
@click.help_option("-h", "--help")
@click.option(
    "-i",
    "--input",
    type=click.Path(exists=True, file_okay=True, dir_okay=False),
    required=True,
    help="Path to the input Zarr file.",
)
@click.option(
    "-c",
    "--csv",
    type=click.File("r"),
    required=True,
    help="Path to the CSV file containing well names.",
)
def rename_wells_cli(csvfile, zarrfile):
    rename_wells(csvfile, zarrfile)
```

Reply:
I like the idea of having a python-accessible way of modifying with a csv, but I will suggest swapping the names. 

The click-wrapped function is what you type at the command line, so I would suggest swapping `rename_wells` and `rename_wells_cli`.